project/Dependencies.scala

```diff
@@ -786,6 +789,7 @@ object Dependencies {
     akkaManagementClusterHttp,
     akkaProtobuf_v3,
     akkaTestkit          % Test,
+    akkaPersistenceTestkit % Test,
```

Comment:
`cluster-core` and other modules don't use `akkaPersistenceTestkit` in their tests.

Reply:
You're right, I've removed them in the new commit. Thanks!